joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller.hpp

```diff
@@ -179,39 +179,39 @@ class JointTrajectoryController : public controller_interface::ControllerInterfa
   rclcpp::Duration action_monitor_period_ = rclcpp::Duration(RCUTILS_MS_TO_NS(50));
 
   // callbacks for action_server_
-  rclcpp_action::GoalResponse goal_callback(
+  JOINT_TRAJECTORY_CONTROLLER_PUBLIC rclcpp_action::GoalResponse goal_callback(
```

Comment:
Please try to make it look nice at least, the visibility macros should be out of the way of the actual code.
```suggestion
  JOINT_TRAJECTORY_CONTROLLER_PUBLIC
  rclcpp_action::GoalResponse goal_callback(
```